e2etest/l2tests/interface_selector.go

```diff
@@ -173,5 +175,213 @@ var _ = ginkgo.Describe("L2-interface selector", func() {
 				}, 10*time.Second, 1*time.Second).Should(gomega.Not(gomega.HaveOccurred()))
 			}
 		})
+
+		ginkgo.It("Specify not existed interfaces", func() {
+			nonExistedInterface := "ens10"
+			for i := range NodeNics {
+				if NodeNics[i] == nonExistedInterface {
+					framework.Fail("Specify not existed interfaces error, Cluster nodes have the interface named ens10")
+				}
+			}
+			resources := internalconfig.ClusterResources{
+				L2Advs: []metallbv1beta1.L2Advertisement{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "with-interfaces",
+						},
+						Spec: metallbv1beta1.L2AdvertisementSpec{
+							Interfaces: []string{nonExistedInterface},
+						},
+					},
+				},
+			}
+			err := ConfigUpdater.Update(resources)
+			framework.ExpectNoError(err)
+
+			svc, _ := service.CreateWithBackend(cs, f.Namespace.Name, "lb-service")
+			defer func() {
+				err := cs.CoreV1().Services(svc.Namespace).Delete(context.TODO(), svc.Name, metav1.DeleteOptions{})
+				framework.ExpectNoError(err)
+			}()
+
+			ingressIP := e2eservice.GetIngressPoint(&svc.Status.LoadBalancer.Ingress[0])
+			err = mac.FlushIPNeigh(ingressIP, executor.Host)
+			framework.ExpectNoError(err)
+
+			for i := range LocalNics {
+				gomega.Eventually(func() error {
+					return mac.RequestAddressResolutionFromIface(ingressIP, LocalNics[i], executor.Host)
+				}, 10*time.Second, 1*time.Second).Should(gomega.HaveOccurred())
+			}
+		})
+
+		ginkgo.It("Address pool connected with two L2 advertisements", func() {
+			svc, _ := service.CreateWithBackend(cs, f.Namespace.Name, "lb-service")
+			defer func() {
+				err := cs.CoreV1().Services(svc.Namespace).Delete(context.TODO(), svc.Name, metav1.DeleteOptions{})
+				framework.ExpectNoError(err)
+			}()
+
+			ingressIP := e2eservice.GetIngressPoint(&svc.Status.LoadBalancer.Ingress[0])
+			resources := internalconfig.ClusterResources{}
+
+			for i := range NodeNics {
+				l2Adv := metallbv1beta1.L2Advertisement{
```

Comment:
what is the purpouse of this tests? It's creating two identical l2advertisements

Reply:
this e2e is used to test the same ip address pool connected with two different L2 Advertisement. the Interfaces field in L2Advertisement should be NodeNics[i]